### DISCUSSION TITLE
What is "altitude" on the right side of the tide graph?

### DISCUSSION THREAD

**Discussion by aukanai** [jaybo/deepzoom-blog#11] [General]
Jaybo,
Could you tell me what the altitude is in reference to on the graph presentation of currents?
I teach a basic boater safety class and often show Deepzoom as a valuable web site. Am blown away by all the new features. Thanks for your work on this.
Bill

**Comment by jaybo** [comment]
Hi Bill,
The altitude refers to the height of the moon or sun in the sky. Probably not something most people are that interested in, but it just helps to visualize where those bodies are at any given moment, since they are ultimately controlling the tides and currents.
Jay
________________________________
From: aukanai ***@***.***>
Sent: Wednesday, October 19, 2022 12:06:52 PM
To: jaybo/deepzoom-blog ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [jaybo/deepzoom-blog] question about depzoom (Discussion #11)


Jaybo,
Could you tell me what the altitude is in reference to on the graph presentation of currents?
I teach a basic boater safety class and often show Deepzoom as a valuable web site. Am blown away by all the new features. Thanks for your work on this.
Bill

**Reply by aukanai** [reply]
Thank you, someone in the class asked the question and now I can answer it. Much appreciated

**Reply by aukanai** [reply]
Jaybo, not to beat this issue to death but have been doing some reading on the subject. Am I accurate in saying, the value of knowing the altitude would be that generally speaking the greater the angle (90°) the milder the tides and hence the currents?

**Reply by jaybo** [reply]
If the earth was a perfect sphere without landforms, predicting tides and currents would be easy.  It's the presence of landforms that complicates predictions; nearby tide stations can have wildly different values.  Each tide or current is unique and is predicted by a series of 20 (or so) harmonic values.  But there are a few generalities we can make:

- tides nearer the equator are generally smaller than those at high latitudes
- when the moon and sun are on the same side of the earth, or on directly opposite sides, the magnitude of the tide will generally be higher
- the best way I've found to visualize tides is to imagine a a bulge of water being pulled toward the moon and sun in the sky.  The bulge is stationary as the earth rotates within the bulge of water.  
- there is a delay (because of the drag effects on the moving water) so that the bulge isn't synchronous with the position of the sun and moon in the sky.  The delay varies with location.  In parts of Puget Sound, for instance, the delay is something like 3 hours.  In the example below, the sun and moon are both (roughly) over Puget Sound at 12:30pm, but the high tide at Naval Base Whidbey Island doesn't happen until 3:44pm.  In other areas, the delay can be 8 hours or more.
- the *altitude* of the sun or moon doesn't really play a critical role in comparison to the phase of the moon.
- try setting the timebase to one month, or one year, and drag the time slider to see all of these effects in action.

![image](https://user-images.githubusercontent.com/1607323/197926507-33d3770a-c84f-4d13-9ea0-93cc25404d91.png)

**Comment by aukanai** [comment]
Jay,
Thank you for all the help with this. I did get the altitude degrees and
moon-sun relationship figured out. One of the issues I had is that the
class is at night. So when I initially put up your site there was no moon
or sun visible. We are currently in between boats so I haven't used your
site since the upgrade. I should have looked at it before showing it in
class. I like the improvements to the site and will do a better job
presenting it next time.
Best regards,
Bill

**Comment by rbelserene** [comment]
Sometimes my graph shows knots on the left and altitude on the right. 
Why are knots there? What is it showing?
It is a completely different graph than when viewing  the tide height. How do I get back to feet?

**Reply by jaybo** [reply]
The graph shows knots when viewing a current station (arrow), and feet, or meters when displaying a tide station (gold square). To select a station, just click on it on the map.

**Reply by rbelserene** [reply]
Thanks!  That makes perfect sense. I hadn't noticed that my usual tide station wasn't selected.
